.rultor.yml

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (C) 2017 Marvin Herman Froeder (velobr@gmail.com)
```

Comment:
files in this project do not include copyright statements.

(if they did we would want them all to one entity)

Reply:
Ops, my bad.

I just used a template I have for my OSS stuff.